services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/structure/directory/AppConfigurationDirectory.java

```diff
@@ -104,7 +105,9 @@ private void addArchiveIfMessageValid(String archiveName, Message message, Confi
 
     ArchiveOnDisk appConfigurationFile = new ArchiveOnDisk(archiveName);
     appConfigurationFile.addWritable(new FileOnDisk("export.bin", message.toByteArray()));
-    countryDirectory.addWritableToAll(__ -> new AppConfigurationSigningDecorator(appConfigurationFile, cryptoProvider,
-        distributionServiceConfig));
+    countryDirectory.addWritableToAll(ignoredValue ->
+        new AppConfigurationSigningDecorator(appConfigurationFile, cryptoProvider,
+        distributionServiceConfig)
+    );
```

Comment:
can be moved to line 109 as wenn in order to maintain consistency

Reply:
I've now moved the line.